tests/storage/snapshots/test_snapshots.py

```
@@ -264,6 +266,48 @@ def test_restore_same_snapshot_twice(
                     expected_result=expected_output_after_restore(1),
                 )
 
+    @pytest.mark.parametrize(
+        "rhel_vm_name, snapshot_with_content",
```

Ruff (0.15.13)

[warning] Wrong type passed to first argument of `pytest.mark.parametrize`; expected `tuple`

Use a `tuple` for the first argument

(PT006)